packages/plugins/geth/src/devtxs.ts

```diff
@@ -0,0 +1,93 @@
+import { __ } from 'embark-i18n';
+import { Embark, Events, Logger } from "embark";
+import Web3 from "web3";
+import constants from "embark-core/constants.json";
+export default class DevTxs {
+  private embark: Embark;
+  private events: Events;
+  private logger: Logger;
+  private web3?: Web3;
+  private regularTxsInt?: NodeJS.Timeout;
+  constructor(embark: Embark) {
+    this.embark = embark;
+    this.logger = embark.logger;
+    this.events = embark.events;
+  }
+  public async init() {
+    const provider = await this.events.request2("blockchain:client:provider", "ethereum");
+    this.web3 = new Web3(provider);
+
+    const accounts = await this.web3.eth.getAccounts();
+    this.web3.eth.defaultAccount = accounts[0];
+
+    this.registerConsoleCommands();
+  }
+
+  private registerConsoleCommands() {
+    this.embark.registerConsoleCommand({
+      description: __("Toggles regular transactions used to prevent transactions from getting stuck when using Geth and Metamask"),
+      matches: ["devtxs on", "devtxs off", "regularTxs on", "regularTxs off"],
+      usage: "devtxs on/off",
+      process: async (cmd, callback) => {
+        if (cmd.startsWith("regularTxs")) {
+          this.logger.info(__("Deprecation notice: The command 'regularTxs on/off' is now deprecated in favor of 'devtxs on/off' and will be removed in future versions."));
+        }
+        const enable = cmd.trim().endsWith('on');
+        this.logger.info(`${enable ? "Enabling" : "Disabling"} regular transactions...`);
+        if (enable) {
+          try {
+            await this.startRegularTxs();
+          } catch (err) {
+            return callback(err);
+          }
+          const msgEnabled = __("Regular transactions have been enabled");
+          callback(null, msgEnabled);
+          return;
+        }
+        try {
+          this.stopRegularTxs();
+        } catch (err) {
+          return callback(err);
+        }
+        const msgDisabled = __("Regular transactions have been disabled");
+        callback(null, msgDisabled);
+      }
+    });
```

Comment:
Just so that I understand: devtx still need to be enabled explicitly to kick in right? Based on a recent discussion with @iurimatias this used to be happening automatically if the node was in `--dev` mode. Are we no longer doing this implicitly?

Reply:
This was fixed here: https://github.com/embark-framework/embark/pull/2120/files#diff-6e8964d2dd079e2afa92e3707c719a87R55